posthog/api/activity_log.py

```diff
@@ -13,6 +13,7 @@
 from posthog.models import ActivityLog, FeatureFlag, Insight, NotificationViewed, User
 from posthog.models.comment import Comment
 from posthog.models.notebook.notebook import Notebook
+from posthog.session_recordings.session_recording_api import ServerTimingsGathered
```

Comment:
really I guess this should move out of session recordings but I bet we revert this change once we've gathered some data so 🤷

Reply:
oh, it turns out it's not safe to share that anyway, so a little duplication is safer than the wrong abstraction